dependencies/setupFmi4c.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Shell script building Hopsan dependency fmi4c
+
+basedir=$(pwd)
+
+name=fmi4c
+codedir=${basedir}/${name}-code
+builddir=${basedir}/${name}-build
+installdir=${basedir}/${name}
+
+
+# Download and verify
+./download-dependencies.py ${name}
+
+# Include general settings
+source setHopsanBuildPaths.sh
+
+# Create build dir and enter it
+mkdir -p $builddir
+cd $builddir
+
+# Generate makefiles
+cmake -Wno-dev -G %HOPSAN_BUILD_CMAKE_GENERATOR% -DCMAKE_INSTALL_PREFIX=${installdir} ${codedir}
```

Comment:
`-G %HOPSAN_BUILD_CMAKE_GENERATOR%` is copied from Windows, we do not use HOPSAN_BUILD_CMAKE_GENERATOR on linux (but we really should) but I must add it for all scripts, I can do that later, remove it for now